### ISSUE TITLE
profile switch every 10 minutes

### ISSUE THREAD

**Issue by ObstKur** [MilosKozak/AndroidAPS#1606]
Reporting bugs
--------------
- **Note the precise time the problem occurred** and describe the circumstances and steps that caused
  the problem
- Note the Build version (found in the About dialog in the app, when pressing the three dots in the
  upper-right corner).
- Obtain the app's log files, which can be found on the phone in
  _/storage/emulated/0/Android/data/info.nightscout.androidaps/_
  See https://github.com/MilosKozak/AndroidAPS/wiki/Accessing-logfiles
- Open an issue at https://github.com/MilosKozak/AndroidAPS/issues/new

**Comment by ObstKur** [comment]
Sorry, I am new here...

Since the last profile switch about 1 hour ago, aaps is constantly updating the basal profile in pump thereby switching between the old basal rate and the new basal rate. I changed basal in ns an did a profile switch withe the same ns profile but changed basal rate. This happens about every five minutes and each time the pump vibrates, which is very annoying. I had this once before and had to deinstall aaps because sleep was not possible anymore with the ever vibrating pump. So I deleted everything and started from scratch. I am now in objective 2. The last time this happened, I was in objective 4. I have exactly one ns-profile which I wanted to modify. Using local profiles does not help, the issue remains.

Build version:
c7ab2ae47-2019.01.19-10:20, fullrelease, nsv 0.10.3-master-20180805

log file:
[AndroidAPS.log](https://github.com/MilosKozak/AndroidAPS/files/2790546/AndroidAPS.log)
